### ISSUE TITLE
Pretty Print Failing Line/Column

### ISSUE THREAD

**Issue by seldridge** [freechipsproject/firrtl#1598]
tl;dr: Use `ir.Info` information to try to pretty print failing line/column from original source.

### Checklist

- [x] Did you write out a description of the feature you want to see?
- [x] Did you look around for any related features?
- [x] Did you specify relevant external information?

### Feature Description

Consider the following circuit that will error in FIRRTL because the width of `out` cannot be determined.

```scala
class Foo extends MultiIOModule {
  val out = IO(Output(SInt()))
  out := DontCare
}
```

This currently errors with:

```
java.lang.ExceptionInInitializerError
	at Main$.<init>(main.scala:16)
	at Main$.<clinit>(main.scala)
	at Main.main(main.scala)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at sbt.Run.invokeMain(Run.scala:115)
	at sbt.Run.execute$1(Run.scala:79)
	at sbt.Run.$anonfun$runWithLoader$4(Run.scala:92)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at sbt.util.InterfaceUtil$$anon$1.get(InterfaceUtil.scala:10)
	at sbt.TrapExit$App.run(TrapExit.scala:257)
	at java.lang.Thread.run(Thread.java:748)
Caused by: firrtl.passes.CheckWidths$UninferredWidth:  : Uninferred width for target below.serialize}. (Did you forget to assign to it?)
    circuit Foo:
    └── module Foo:
        └── out
```

This is using `Target` information to pretty print the failing target. This probably isn't the best and we should just use the source locator stored in the `Info` field to show line/column (I realize I added the target pretty printing error...).  

Ideally, I'd like to see:

```
[error] firrtl.passes.CheckWidths$UninferredWidth:  : Uninferred width for target 
[error] In file Foo.scala:
[error]   val out = IO(Output(SInt()))
[error]   ^
```

### Type of Feature

- documentation                     

- new feature/API

### Related Features

No.

### External Information

None.
